src/GraphQL/Schemas/Primary/Mutations/AddToTrackedMutation.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Enjin\Platform\GraphQL\Schemas\Primary\Mutations;
+
+use Closure;
+use Enjin\Platform\Enums\Global\ModelType;
+use Enjin\Platform\GraphQL\Schemas\Primary\Traits\InPrimarySchema;
+use Enjin\Platform\Interfaces\PlatformGraphQlMutation;
+use Enjin\Platform\Models\Syncable;
+use Enjin\Platform\Rules\MinBigInt;
+use Enjin\Platform\Services\Database\CollectionService;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Support\Mutation;
+
+class AddToTrackedMutation extends Mutation implements PlatformGraphQlMutation
+{
+    use InPrimarySchema;
+
+    /**
+     * Get the mutation's attributes.
+     */
+    public function attributes(): array
+    {
+        return [
+            'name' => 'AddToTracked',
+            'description' => __('enjin-platform::mutation.add_to_tracked.description'),
+        ];
+    }
+
+    /**
+     * Get the mutation's return type.
+     */
+    public function type(): Type
+    {
+        return GraphQL::type('Boolean!');
+    }
+
+    /**
+     * Get the mutation's arguments definition.
+     */
+    public function args(): array
+    {
+        return [
+            'type' => [
+                'type' => GraphQL::type('ModelType!'),
+                'description' => __('enjin-platform::mutation.add_to_tracked.args.model_type'),
+            ],
+            'chainIds' => [
+                'type' => GraphQL::type('[BigInt!]!'),
+                'description' => __('enjin-platform::mutation.add_to_tracked.args.chain_ids'),
+            ],
+            'hotSync' => [
+                'type' => GraphQL::type('Boolean'),
+                'description' => __('enjin-platform::mutation.add_to_tracked.args.hot_sync'),
+                'defaultValue' => true,
+            ],
+        ];
+    }
+
+    /**
+     * Resolve the mutation's request.
+     */
+    public function resolve($root, array $args, $context, ResolveInfo $resolveInfo, Closure $getSelectFields, CollectionService $collectionService): mixed
+    {
+        collect($args['chainIds'])->each(function ($id) use ($args, $collectionService) {
```

Comment:
Do I need to update this PR https://github.com/enjin/platform-multi-tenant/pull/51/files so that when linking existing data to the user, the collection will also be indexed?